lib/gis/parser_rest_md.c

```diff
@@ -51,7 +53,15 @@ void usage_rest_md(bool rest)
     if (!st->pgm_name)
         st->pgm_name = "??";
 
-    /* main header */
+    /* print metadata used by man/build*.py */
+    fprintf(stdout, "---\n");
+    fprintf(stdout, "module: %s\n", st->pgm_name);
+    fprintf(stdout, "description: %s\n", st->module_info.description);
```

Comment:
Do have a top-level label, just like for options? I didn't check, but I would say yes.

Reply:
Do you suggest to add label to metadata as a new line or use label value as description if description is not defined?

Reply:
@wenzeslaus I suggest to keep it for seperate PR.